introduction.md

```diff
@@ -0,0 +1,3 @@
+reviewing it, so when got asked I still remember.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
just trying to review all the things I did in the past. codeschool is good at git and js and lacking at .NET, pl is good at .NET but lacking CS grading